test/markups/README.man

```diff
@@ -0,0 +1,6 @@
+.Dd $Mdocdate: March 23 1981 $
+.Dt README 1
+.Os GitHub
+.Sh README.man
```

Comment:
This is an mdoc(7) syntax error, which is probably not a good thing to have in an example/test file. The first section header in a manual page MUST be exactly ".Sh NAME". Your line is also invalid because it contains lower case letters.

Reply:
And after that it should provide data for whatis(1)'s scraping; see lexgrog(1)'s _WHATIS PARSING_ as an example.

    .Sh NAME
    .Nm foo
    .Nd program to do something